classes/models/submission.php

```diff
@@ -424,11 +424,8 @@ public function get_file_annotations() {
         $fs = new file_storage();
 
         $annotatedfiles = [];
-        foreach (
-            $fs->get_area_files($this->get_context_id(), 'mod_coursework', 'submissionannotations', $this->id)
-            as
-            $file
-        ) {
+        foreach ($fs->get_area_files($this->get_context_id(), 'mod_coursework', 'submissionannotations', $this->id)
+        as $file) {
```

Comment:
the line split here I think is one of the more recently introduced Moodle coding standard